src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.exception.DeleteOutOfRangeException;
+import duke.exception.DoneOutOfRangeException;
+import duke.exception.EmptyDeadlineException;
+import duke.exception.EmptyEventException;
+import duke.exception.EmptyTodoException;
+import duke.exception.MissingDeadlineDateException;
+import duke.exception.MissingDeleteArgumentException;
+import duke.exception.MissingDoneArgumentException;
+import duke.exception.MissingEventDateException;
+import duke.exception.MissingFindArgumentException;
+import duke.exception.UnknownCommandException;
+
+/**
+ * Duke Bot is a program that c an schedule tasks and mark them as done, delete
+ * and save them on a file.
+ *
+ * @author Lim Jin Feng
+ * @version 0.1
+ * @since   2020-08-20
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs Duke Bot in a given file path.
+     *
+     * @param filePath  Where Duke Bot will save it's tasks.
+     */
+    public Duke(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.tasks = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            this.ui.sendFailedInitialiseMessage();
```

Comment:
Instead of specifically catching exceptions as part of your flow (in this case, to check if the file exists), we should manually refactor this part of the code to check for file existance, or even better, just use `.getParentFile().mkdirs()` and/or `.createNewFile()`, which are built in Java API methods that creates the files if they don't exist. Exceptions should be meant for error and ...exceptional cases.